app.py

```diff
@@ -1,13 +1,19 @@
 from flask import Flask, render_template, request, redirect, url_for
 import session_items as session
+import requests
```

Comment:
This dependency needs to be added to `requirements.txt` (are you running `setup.sh`? from the README instructions?)

Reply:
Got it. I've updated the requirements.txt with requests==2.23.0
Good point.